.ci/deploy-linux.sh

```diff
@@ -44,13 +49,25 @@ if [ "$DEPLOY_APPIMAGE" = "true" ]; then
     mv ./RPCS3*.AppImage "$RPCS3_APPIMAGE"
 
     # If we're building using Azure Pipelines, let's copy over the AppImage artifact
-    if [ -n "$BUILD_ARTIFACTSTAGINGDIRECTORY" ]; then
-        cp "$RPCS3_APPIMAGE" ~/artifacts
+    if [ "$CIRRUS_CI" = "true" ]; then
```

Comment:
first check for BUILD_ARTIFACTSTAGINGDIRECTORY and then check for CIRRUS_CI.
saves one or two lines xD